content/guides/composite_monitors.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Guide to Composite Monitors
+kind: guide
+listorder: 9
+---
+
+***If you're unfamiliar with the basics of Datadog Monitors, first read the [Guide to Monitors](/guides/monitors)***
+
+---
+
+Composite monitors let you combine many individual monitors into one so that you can define more specific alert conditions. Choose two or more existing monitors - monitor A and monitor B, say - and then set a trigger condition using boolean operators (e.g. “A && B”). The composite monitor will trigger when its individual monitors' statuses simultaneously have values that cause the composite's trigger condition to be true.
+
+After you create a composite monitor, you can modify it just as you would an individual monitor. To change the trigger condidtion, edit the composite monitor. To change one of its constituent monitors, though - e.g. to disable its notifications in favor of the composite notification - simply edit that individual monitor.
+
+## Creating composite monitors
+
+In Datadog, go to the New Monitor page and select 'Composite Monitor' from the list of monitor types:
+
+![choose-composite-type](/static/images/composite_monitors/select-monitor-type.png)
+
+### Choose individual monitors
+
+You can choose up to 10 individual monitors to use in the new composite monitor. When choosing monitors, the following rules apply:
+
+* No individual monitor may itself be a composite monitor
```

Comment:
@jyee I put the chosen-monitor requirements in a list -- good idea, I think it's clearer.

Does it make more sense than before. Still need Eng to review it for correctness.